Cargo.toml

```diff
@@ -441,7 +441,6 @@ openssl = { version = "0.10", features = ["vendored"] }
 # To make most examples faster to compile, we split out assets and http-related stuff
 # This trims off like 270 dependencies, leading to a significant speedup in compilation time
 [features]
```

Comment:
I'd rather the default behavior of `cargo run --example todomvc` to not be a runtime error - is there any reason why we can't keep "desktop" as the defualt?